bucket/resticprofile.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "0.23.0",
+    "description": "Configuration profiles manager and scheduler for restic backup",
+    "homepage": "https://creativeprojects.github.io/resticprofile/",
+    "license": "GPL-3.0-only",
+    "suggest": {
+        "Restic and/or Rustic" : [ "restic", "rustic" ]
```

Comment:
```suggestion
        "Restic": "restic",
        "Rustic": "rustic"
```

Reply:
I don't think that's right. resticprofile needs one of restic or rustic installed. It's possible to have both, but usually exactly one of them will be used. That means if restic is installed rustic isn't needed and vice versa.

Currently rustic has less windows support. So for scoop restic is the preferred choice. That was the reason I originally used "depends" instead of "suggest".

Reply:
Then just suggest restic, you can omit rustic

Reply:
OK. Done. We can readd rustic as an alternative for restic later, if requested by our users.